ros_buildfarm/templates/release/release_reconfigure-jobs_job.xml.em

```diff
@@ -13,6 +13,16 @@
     priority=30,
 ))@
 @(SNIPPET(
+    'property_parameters-definition',
+    parameters=[
+        {
+            'type': 'boolean',
+            'name': 'dry_run',
```

Comment:
Almost all current parameters are lower case. Therefore I will stick to it in this PR. If we think its worth we can update the existing code in a separate PR though.